src/store/TodosReducer.js

```diff
@@ -11,6 +11,10 @@ const todosReducer = (state = INITIAL_STATE, action) => {
       return {
           todos: [...todos, action.todo],
       }
+    case 'SHOW_TODOS':
+      return {
+        todos: todos.map(todo => todo.text),
+      }
```

Comment:
Visto que o que a API devolve é um array de elementos temos que atualizar esta lista.

todos = [ {id: '1', text: 'something' } ]

```suggestion
      return {
        todos:  action.todo,
      }
```